testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-oauth2-https-proxy-authn/config.properties

```diff
@@ -0,0 +1,30 @@
+node.id=will-be-overwritten
+node.environment=test
+
```

Comment:
please move files related to trino to `trino` directory:
`singlenode-oauth2-https-proxy-authn/trino/config.properties`